html/stipendier.html

```diff
@@ -1,59 +1,115 @@
-                <li>Meny</li> <!-- Drop down menu here -->
-            </ul>
-
-            <a href="#"><img src="#" alt="svenska">Sv</a>
-            <a href="#"><img src="#" alt="english">En</a>
+        <ul>
+          <li><a href="html/om_oss.html">Om oss</a></li>
+          <li><a href="html/konstnarsguiden.html">Konstnärsguiden</a></li>
+          <li><a href="#">Vanliga frågor</a></li>
+          <li><a href="html/kontakt.html">Kontakt</a></li>
+          <li>Meny</li>
+          <!-- Drop down menu here -->
+        </ul>
 
-            <a href="#">Logga in</a>
+        <a href="#"><img src="#" alt="svenska" />Sv</a>
+        <a href="#"><img src="#" alt="english" />En</a>
 
-        </nav>
+        <a href="#">Logga in</a>
+      </nav>
     </header>
 
     <main>
-
-        <ul class="breadcrumbs">
-            <li>Startsida / </li>
-            <li></li>
-        </ul>
-
+      <ul class="breadcrumbs">
+        <li><a href="/index.html"> Startsida /</a></li>
+        <li>Stipendier & bidrag</li>
+      </ul>
+      <section>
+        <h1>Stipendier & bidrag</h1>
+        <p>
+          Konstnärsnämnden fördelar ett brett spektrum av stipendier och bidrag
+          till yrkesverksamma konstnärer inom bild, form, musik, teater, dans
+          och film. Stödet riktar sig främst till dig som är enskild frilansande
+          konstnär, bosatt i Sverige och/eller med din konstnärliga verksamhet
+          här. Under 2019 beslutade Konstnärsnämnden om stipendier och bidrag
+          uppgående till 179 miljoner kronor. Kulturbryggans beslut ingår i
+          beloppet.
+        </p>
+      </section>
+      <section>
+        <h2>Här kan du söka stipendier och läsa om ansökningstider.</h2>
+        <a href="#">Ansök här</a>
+      </section>
+      <section>
+        <p>
+          Konstnärsnämnden beslutar årligen om cirka 85 miljoner kronor till
+          konstnärer inom bild och form. Bidragen till konstnärer inom musik
+          uppgår till 31 miljoner kronor per år. Inom dansområdet beviljas 13
+          miljoner, filmområdet 12 miljoner kronor och teaterområdet 10
+          miljoner. Inom området dans behandlar myndigheten även ansökningar
+          från cirkusartister. Dramatiker får stöd av
+          <a href="https://www.svff.se/">Sveriges författarfond</a>.
+          Institutioner och fria grupper kan söka bidrag hos
+          <a href="https://www.kulturradet.se/">Statens Kulturråd</a>.
+        </p>
+        <h2>Cirka 1 800 stipendier delas ut per år</h2>
+        <p>
+          Varje år behandlar myndigheten cirka 7 000 ansökningar från verksamma
+          konstnärer. Omkring 1 800 ansökningar beviljas varje år. De sökande
+          tilldelas stipendium eller bidrag i någon form. Grundkriteriet är
+          konstnärlig kvalitet och ekonomiskt behov. Vi strävar även efter
+          mångfald och spridning av genrer, ålder, kön och geografisk
+          fördelning.
+        </p>
```

Comment:
Maybe this can be marked up further? I think each of these could make an article. I know you discussed this a bit earlier today, wether to use an article or not. Let's look at it together at friday meeting

Reply:
i have made some changes. Hope it looks better now